books/RayTracingInOneWeekend.html

```diff
@@ -483,10 +483,11 @@
 <div class='together'>
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
 calculating whether a ray hits a sphere is pretty straightforward. Recall that the equation for a
-sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 = R^2$. The way you can read that
-equation is “for any $(x, y, z)$, if $x^2 + y^2 + z^2 = R^2$ then $(x,y,z)$ is on the sphere, and
-otherwise it is not”. It gets uglier if the sphere center is at
-$(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 = R^2$. Put another way, for
+$(x,y,z)$ _inside_ the sphere, $x^2 + y^2 + z^2 < R^2$, for points _outside_ the sphere, $x^2 + y^2
```

Comment:
"for any point $(x,y,z)$ _inside_ the sphere ..."
Maybe
"for a point $(x,y,z)$ ...."

Reply:
It's grammatically incorrect (and I argue confusing) to say:
for coordinate inside sphere, for points outside sphere, for points on sphere.

This should be made complete, e.g.
for point inside the sphere, for points outside the sphere, for points on he sphere...

Reply:
I have a slight preference for flipping the subject and object in the sentence:

"Put another way, for any point $(x,y,z)$, the point is inside the sphere if ..., the point is outside the sphere if ..., the point is on the sphere if ...."